shell/bundle.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -ue
+#!/usr/bin/env bash -ue
```

Comment:
Be really careful with multiple arguments here -- this may reduce portability on soe systems; https://stackoverflow.com/questions/4303128/how-to-use-multiple-arguments-for-awk-with-a-shebang-i-e/4304187#4304187